EventFilter/CTPPSRawToDigi/src/RawToDigiConverter.cc

```diff
@@ -280,14 +282,15 @@ void RawToDigiConverter::Run(const VFATFrameCollection &coll, const TotemDAQMapp
     if (record.status.isOK())
     {
       const VFATFrame *fr = record.frame;
-      DiamondVFATFrame *diamondframe = (DiamondVFATFrame*) fr;
+      DiamondVFATFrame *diamondframeTmp = (DiamondVFATFrame*) fr;
+      DiamondVFATFrame diamondframe(*diamondframeTmp);
```

Comment:
is the full copy really necessary?
The use case below suggests that it is not needed 
and that this file should have no modifications.